hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/ConstantSizeRegionSplitPolicy.java

```diff
@@ -94,4 +94,33 @@ long getDesiredMaxFileSize() {
   public boolean positiveJitterRate() {
     return this.jitterRate > 0;
   }
+
+  /**
+   * @return true if region size exceed the sizeToCheck
+   */
+  protected boolean isExceedSize(long sizeToCheck, String extraLogStr) {
+    if (overallHregionFiles) {
+      long sumSize = 0;
+      for (HStore store : region.getStores()) {
+        sumSize += store.getSize();
+      }
+      if (sumSize > sizeToCheck) {
```

Comment:
We should just return this comparison and let each caller decide how to log it? That would discard the need for having an extra param just for the sake of logging.

Reply:
The extraLogStr exactly seems a bit strange, but there are several variables used by logging which calculated inside isExceedSize, if we want remove extraLogStr, I think there are some other ways.
1. Use a map as param instead of extraLogStr.
2. Change the return value from boolean to other obj such as map.
3. Make the logging more simpler.

I prefer the current way, WDYT?
Thanks. @wchevreuil

Reply:
We are breaking isolation and cohesion principles, this method should log only the info it has locally, without requiring any additional param just for logging. The additional info available in eventual callers should then be logged by those callers, not passed through as params, here I suggest the log to be only:

`LOG.debug("ShouldSplit because region size is big enough "
            + "size={}, sizeToCheck={}", StringUtils.humanSize(sumSize),
          StringUtils.humanSize(sizeToCheck))`

Then on IncreasingToUpperBoundRegionSplitPolicy you log the extra info on an additional debug message:

`LOG.debug(regionsWithCommonTable={}", tableRegionsCount)`

Reply:
Yeah, i am agree with you, will fix later.
Thanks.